zulip/integrations/bridge_with_slack/bridge_with_slack_config.py

```diff
@@ -0,0 +1,14 @@
+config = {
+    "zulip": {
+        "email": "zulip-bot@email.com",
+        "api_key": "put api key here",
+        "site": "https://chat.zulip.org",
+        "stream": "test here",
+        "topic": "<- slack-bridge",
```

Comment:
It would be nice if this could be a mapping between Zulip stream+topic and Slack channels. A single bridge could be used to bridge multiple Slack channels.

Reply:
This could be made such that if "topic" is None, then the entire channels are mapped to each topic in a stream. But this takes a while to make, and better done once the current version has been merged.

Reply:
Sounds reasonable, for a start :+1: 
Though, I could imagine wanting each channel in Slack being mapped to different topics in different streams.